libse/Forms/FixCommonErrors/EmptyFixCallback.cs

```diff
@@ -68,4 +69,19 @@ public HashSet<string> GetAbbreviations()
         }
 
     }
+
+    public class CustomFixCallback : EmptyFixCallback
```

Comment:
Add new new fix-callbacks which extends **EmptyFixllCallback** because we won't be able to check if word is name if we use **EmptyFixCallback** 'cause it always return **false** if we call **IsName(candidate)**